.github/workflows/template-validation.yaml

```
@@ -0,0 +1,81 @@
+name: Validate AZD template
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    - cron: '0 0 * * *' # Run at midnight every day
+
+jobs:
+  infra:
+    name: "Infra Biceps Validation"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

Checkout action is updated to V4

```suggestion
        uses: actions/checkout@v4
```